python/lsst/pipe/tasks/processCcdWithFakes.py

```diff
@@ -445,6 +474,39 @@ def run(self, fakeCat, exposure, wcs=None, photoCalib=None, exposureIdInfo=None,
         resultStruct = pipeBase.Struct(outputExposure=exposure, outputCat=sourceCat)
         return resultStruct
 
+    def composeFakeCat(self, fakeCats, skyMap):
+        """Concatenate the fakeCats from tracts that may cover the exposure.
+
+        Parameters
+        ----------
+        fakeCats : `list` of `lst.daf.butler.DeferredDatasetHandle`
+            Set of fake cats to concatenate.
+        skyMap : `lsst.skymap.SkyMap`
+            SkyMap defining the geometry of the tracts and patches.
+
+        Returns
+        -------
+        combinedFakeCat : `pandas.DataFrame`
+            All fakes that cover the inner polygon of the tracts in this
+            quantum.
+        """
+        if len(fakeCats) == 1:
+            return fakeCats[0].get(
+                datasetType=self.config.connections.fakeCats)
+        outputCat = []
+        for fakeCatRef in fakeCats:
+            cat = fakeCatRef.get(
+                datasetType=self.config.connections.fakeCats)
+            tractId = fakeCatRef.dataId["tract"]
+            # Make sure all data is within the inner part of the tract.
+            outputCat.append(cat[
+                skyMap.findTractIdArray(cat[self.config.insertFakes.raColName],
+                                        cat[self.config.insertFakes.decColName],
+                                        degrees=False)
```

Comment:
@morriscb Same comment as above: `self.config.insertfakes.raColName` and `self.config.insertFakes.decColName` should be changed to `self.config.ra_col` and `self.config.dec_col` to remove deprecated column names.